src/hooks/useMediaQuery/__stories__/useMediaQuery.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { useRef, useCallback, useState } from "react";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import useMediaQuery from "..";
+import "../../__stories__/general-hooks-stories.scss";
+
+<Meta title="Hooks/useMediaQuery" />
+
+# useMediaQuery
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Feedback](#feedback)
+
+## Overview
+
+This hook helps maps if a media query is matched or not
+
+<Canvas>
+  <Story name="Single Rule">
+    {() => {
+      const [mediaQueryIsMatching] = useMediaQuery(["screen and (max-width: 1023px) and (min-width: 768px)"]);
```

Comment:
cool! not sure if string API is better then more structure API( attribute for media type, attribute for media features)